decentraland/unity-renderer
# feat: add `analyticsKernelService` with renderer-protocol

## What does this PR change?

Use `renderer-protocol` instead of JSON-based communication

## Test

- Open https://play.decentraland.zone/?renderer-branch=feat/analytics&kernel-branch=feat/analytics
- Check if PerformanceReport, SystemInfoReport, AnalyticsEvent, SetDelightedSurveyEnabled, works

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
